diff -up vlmc-13c4dbc/src/CMakeLists.txt.ldf vlmc-13c4dbc/src/CMakeLists.txt
--- vlmc-13c4dbc/src/CMakeLists.txt.ldf	2010-12-14 22:04:54.928224348 +0100
+++ vlmc-13c4dbc/src/CMakeLists.txt	2010-12-14 22:06:11.991628249 +0100
@@ -360,7 +360,7 @@ ELSE(NOT WITH_GUI)
     ENDIF( APPLE )
 
     ADD_EXECUTABLE( vlmc ${GUI_TYPE} ${VLMC_SRCS} ${VLMC_MOC_SRCS} ${VLMC_UIS_H} ${VLMC_RCC_SRCS} )
-    TARGET_LINK_LIBRARIES( vlmc ${VLMC_LIBS} )
+    TARGET_LINK_LIBRARIES( vlmc ${VLMC_LIBS} -lX11 )
 
 ENDIF( NOT WITH_GUI )
 
