--- ./src/Project/WorkspaceWorker.cpp.orig	2018-03-08 01:01:36.043866949 +0000
+++ ./src/Project/WorkspaceWorker.cpp	2018-03-08 01:02:14.669251802 +0000
@@ -24,6 +24,8 @@
 # include "config.h"
 #endif
 
+#include <unistd.h> // link
+
 #include "Project/WorkspaceWorker.h"
 
 #include "Media/Media.h"
diff -up vlmc-20120408/src/vlmc.h.gcc47 vlmc-20120408/src/vlmc.h
--- vlmc-20120408/src/vlmc.h.gcc47	2012-03-29 19:37:10.000000000 +0200
+++ vlmc-20120408/src/vlmc.h	2012-04-08 17:33:15.322088791 +0200
@@ -24,6 +24,8 @@
 #ifndef VLMC_H
 #define VLMC_H
 
+#include <unistd.h> // sleep
+
 #include <QtGlobal>
 
 #ifdef Q_OS_WIN
